backend/src/types.ts

```diff
@@ -1215,3 +1215,8 @@ export type ResponseStatus = {
   success: boolean;
   error: string;
 };
+
+export enum ServiceAddressAnnotation {
+  EXTERNAL_REST = 'routing.opendatahub.io/external-address-rest',
+  EXTERNAL_GRPC = 'routing.opendatahub.io/external-address-grpc',
```

Comment:
Are we expecting connecting the proxy server through grpc? I know between servers can be done, but since it's a proxified call browsers don't support grpc natively

Reply:
We aren't currently using that annotation, I just added it in the constant here because it was added at the same time on the service in case we may need it later. Maybe I should just remove it?

Reply:
I think you can keep it because it's in the generic types folder -- but I agree with Lucas, there is not a likelihood of this working through the http-proxy -- I think Pipelines MLMD uses envoy proxy which is a http endpoint that converts it... @Gkrumbach07 would have to confirm that though.